app/controllers/oaisys/pmh_controller.rb

```diff
@@ -120,8 +119,9 @@ def public_items_for_metadata_format(verb:, format:, page:, restricted_to_set: n
     model = model.created_on_or_after(from_date) if from_date.present?
     model = model.created_on_or_before(until_date) if until_date.present?
 
-    model = model.page(page).per(ITEMS_PER_REQUEST)
-    cursor = (page - 1) * ITEMS_PER_REQUEST
+    items_per_request = Oaisys::Engine.config.items_per_request
+    model = model.page(page).per(items_per_request)
+    cursor = (page - 1) * items_per_request
```

Comment:
Do we need to do any sanity checking on `page`? It looks like it comes straight from the API arguments, so if someone passed a 0 there, cursor here would turn into a negative offset?

Reply:
Page comes from the resumptionToken which it's assumed no one will modify. I've added a check to ensure that page is always greater than 0, throwing a BadResumptionTokenError otherwise

Reply:
Ah right, that makes sense. Good idea to enforce that assumption anyways 👍